src/jwk.rs

```diff
@@ -271,6 +272,46 @@ impl AlgorithmParameters {
             _ => Err(unexpected_key_type_error!(KeyType::Octet, self.key_type())),
         }
     }
+
+    /// RFC 7638
+    pub fn thumbprint(
+        &self,
+        algo: &'static ring::digest::Algorithm,
```

Comment:
Okay!
Do I reexport the module or should I make a small facade?
ring doesn't commit to backwards compatibility, and all we really need to expose are the statics for each algorithm.

Reply:
If it's not too much work, a small facade might be better.

Otherwise, I think re-exporting the module might be the easiest. 

We could also direct the users to the facade/re-exported module via the documentation for this method.